src/pyhf/workspace.py

```diff
@@ -61,15 +72,14 @@ def get_measurement(self, **config_kwargs):
 
         Returns:
             :obj:`dict`: A measurement object adhering to the schema defs.json#/definitions/measurement
+
         """
         m = self._get_measurement(**config_kwargs)
         utils.validate(m, 'measurement.json', self.version)
         return m
 
     def _get_measurement(self, **config_kwargs):
-        """
-        See `Workspace::get_measurement`.
-        """
+        """See `Workspace::get_measurement`."""
```

Comment:
Can we use the `~` syntax to link this?

Reply:
Nevermind. Since the method starts with an underscore this will never show up in the docs.